apps/srt-live-transmit.cpp

```diff
@@ -716,7 +722,7 @@ int main(int argc, char** argv)
                 std::list<std::shared_ptr<MediaPacket>> dataqueue;
                 if (src.get() && src->IsOpen() && (srtrfdslen || sysrfdslen))
                 {
-                    while (dataqueue.size() < 10)
+                    while (dataqueue.size() < (size_t) cfg.buffering)
```

Comment:
If you check the >0 condition using an intermediate variable, you can then as well set `cfg.buffering` type as `size_t`.